README.md

```diff
@@ -62,6 +62,21 @@ mackup restore
 
 Done!
 
+**Warning Mackup won't backup private data by default (ssh keys, bitcoin wallets)**
```

Comment:
No need to be bold and warn, just put a regular sentence?

Reply:
My thinking here was that it needs to be larger text so it jumps out at people. Users will be quite alarmed when it doesn't backup what they expect it to.